### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by FK78**
Resolves #2636, #2637, #2638, #2639, #2640

**Overall change:** _Merging dependabot PRs._

**Code changes:**

- _`package.json`, `package-lock.json` and `CHANGELOG.md` bumped._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by andrewscfc** [comment]
Hey when we bump dependencies do we check what has changed between versions? Just to avoid incidents like: https://blog.npmjs.org/post/180565383195/details-about-the-event-stream-incident 

I guess snyk could pick up this kind of thing?

**Review by andrewscfc — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -96,14 +96,14 @@
     "husky": "^3.0.9",
     "jest": "^24.9.0",
     "jest-fetch-mock": "^2.1.2",
-    "jest-styled-components": "^6.3.3",
+    "jest-styled-components": "^6.3.4",
     "json5": "^2.1.1",
     "lerna": "^3.18.4",
-    "lint-staged": "^9.4.2",
+    "lint-staged": "^9.4.3",
```
Comment (andrewscfc) [inline package.json:102]: Version bumped for security issue in dependency of `lint-staged`: https://github.com/okonet/lint-staged/releases/tag/v9.4.3

File: package.json [hunk 2]
```diff
@@ -83,7 +83,7 @@
     "enquirer": "^2.3.2",
     "eslint": "^6.6.0",
     "eslint-config-airbnb": "^18.0.1",
-    "eslint-config-prettier": "^6.5.0",
+    "eslint-config-prettier": "^6.6.0",
```
Comment (andrewscfc) [inline package.json:86]: Hard to tell what changed between releases: https://github.com/prettier/eslint-config-prettier/releases/tag/v6.6.0

File: package.json [hunk 3]
```diff
@@ -96,14 +96,14 @@
     "husky": "^3.0.9",
     "jest": "^24.9.0",
     "jest-fetch-mock": "^2.1.2",
-    "jest-styled-components": "^6.3.3",
+    "jest-styled-components": "^6.3.4",
```
Comment (andrewscfc) [inline package.json:99]: Minor typescript interface change: https://github.com/styled-components/jest-styled-components/releases/tag/v6.3.4

**Comment by andrewscfc** [comment]
> Hey when we bump dependencies do we check what has changed between versions? Just to avoid incidents like: https://blog.npmjs.org/post/180565383195/details-about-the-event-stream-incident
> 
> I guess snyk could pick up this kind of thing?

Just chatted with @FK78 about this, I've approved but I'll broad the conversation in `si_all_tech` about how we do or if we should analyse for vulnerabilities in bumped dependencies.

**Review by jroebu14 — APPROVED** [review]

**Comment by amywalkerdev** [comment]
Snapshots still failing on CI

**Comment by FK78** [comment]
> Snapshots still failing on CI

They are passing locally even after pulling in latest and running `npm run test:unit -- -u`

**Comment by amywalkerdev** [comment]
Hm weird. I'll try it locally as well

**Pull request merged**